huaweicloud/services/meeting/resource_huaweicloud_meeting_conferences.go

```diff
@@ -0,0 +1,1145 @@
+		result[i] = map[string]interface{}{
+			"user_id":        participant.UserUUID,
+			"account_id":     participant.AccountId,
+			"name":           participant.Name,
+			"role":           participant.Role,
+			"type":           participant.Type,
+			"is_auto_invite": participant.IsAutoInvite,
+			"phone":          participant.ID, // The ID is the phone number of the attendee.
+			"email":          participant.Email,
+			"sms":            participant.SMS,
+		}
+	}
+
+	log.Printf("[DEBUG] The participant list result of the conference is %#v", result)
+	return result
+}
+
+func flattenShowAudiencePolicy(subConference conferences.ShowAudiencePolicy) []map[string]interface{} {
+	if reflect.DeepEqual(subConference, conferences.ShowAudiencePolicy{}) {
+		return nil
+	}
+
+	result := []map[string]interface{}{
+		{
+			"mode":                subConference.ShowAudienceMode,
+			"base_audience_count": subConference.BaseAudienceCount,
+			"multiple":            subConference.Multiple,
+		},
+	}
+
+	log.Printf("[DEBUG] The audience display policy result of the subconference is %#v", result)
+	return result
+}
+
+func flattenSubConferenceConfiguration(subConference conferences.SubConfiguration) []map[string]interface{} {
+	if reflect.DeepEqual(subConference, conferences.SubConfiguration{}) {
+		return nil
+	}
+
+	result := []map[string]interface{}{
+		{
+			"callin_restriction":          subConference.CallInRestriction,
+			"audience_callin_restriction": subConference.AudienceCallInRestriction,
+			"allow_guest_start":           subConference.AllowGuestStartConf,
+			"waiting_room_enabled":        subConference.EnableWaitingRoom,
+			"show_audience_policies":      flattenShowAudiencePolicy(subConference.ShowAudiencePolicy),
+		},
+	}
+
+	log.Printf("[DEBUG] The configuration result of the subconference is %#v", result)
+	return result
+}
+
+func flattenSubConferences(subConferences []conferences.SubConference) []map[string]interface{} {
+	if len(subConferences) < 1 {
+		return nil
+	}
+
+	result := make([]map[string]interface{}, len(subConferences))
+	for i, subConference := range subConferences {
+		mediaType := strings.Split(subConference.MediaType, ",")
+		sort.Strings(mediaType)
+		result[i] = map[string]interface{}{
+			"id":               subConference.ID,
+			"media_types":      mediaType,
+			"start_time":       subConference.StartTime,
+			"end_time":         subConference.EndTime,
+			"is_auto_record":   subConference.IsAutoRecord,
+			"record_auth_type": subConference.RecordAuthType,
+			"subconfiguration": flattenSubConferenceConfiguration(subConference.SubConfiguration),
+		}
+	}
+
+	log.Printf("[DEBUG] The result of the subconference is %#v", result)
+	return result
+}
+
+func flattenConferenceCycleParams(cycleParams conferences.CycleParams) []map[string]interface{} {
+	if reflect.DeepEqual(cycleParams, conferences.CycleParams{}) {
+		return nil
+	}
+
+	// No password return, it is a sensitive parameter.
```

Comment:
remove this comment, there no password in this structure

Reply:
sure, I finished